tests/Unit/LdapUsersTest.php

```diff
@@ -260,6 +262,7 @@ public function test_getUser_UsesCorrectLDAPFilterAndBaseDn()
         $usedBaseDn = null;
         $usedFilter = null;
         $filterBind = null;
+        $memberOfField = Config::getRequiredMemberOfField();
```

Comment:
The changes to this file are unnecessary. Testing against the configured memberOf field value places an unnecessary dependency of the INI config in what is supposed to be an isolated unit test. Further, since the test doesn't set a custom config value, using the configured value just uses the default value. It also makes the test less explicit, and thus less useful. Please remove the changes to this file.